### PR TITLE
Add Minions INU (BEP20)

### PR DESCRIPTION
**Pull request by CryptoDevilJapan**
Adding info for BEP20 token 'Minions INU'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by CryptoDevilJapan** [comment]
⭐Hello TrustWallet team. MinionsINU has gained momentum and is rising day by day. Just 34days old.
The team and community are very active on Twitter Space. We attend at least 3 meetings a day. (Images are shared below as an attachment)
⭐Answers to your questions and additional information are below:
✅Our in-game Stakeable NFTs are ready. We have big goals both in terms of DeFi with Stake and in terms of game and GameFi. We also have a partnership initiative for Betting Platform exclusive to the upcoming football matches.
You can watch on Game Trailer on Youtube: https://www.youtube.com/watch?v=5l--QiF_1k4
You can watch on NFT Marketplace review on Youtube: https://www.youtube.com/watch?v=QzTg8DwZ8mU
✅MinionsINU has a website where all financial details are shared and a Whitepaper, and a clean RoadMap that we have created with realistic steps.
✅We benefit Trust Wallet users by our coming integrations and huge community the majority off them use Trust Wallet.
✅MinionsINU was designed with the concept of Sing2Earn last year. We relaunched to create a new design and a more complex ecosystem and increase financial security.
Just MinionsINU reached 15k+ holders, token distributed to 1768-1893 wallets by airdrop to old holders.( We have an Excel file about it if you want we can share it with you.)
We reach 500-1000 holders daily with CoinGecko and CMC listings, negotiations with Kucoin and aggressive marketing. We think that this figure will play in the 300-1000 range as the exchange listings.
✅We did not hold a Presale
✅We are listed on PancakeSwapV2
⭐Contract Audited and all financial detail shared on Website/Whitepaper&Telegram:
https://github.com/KISHIELD/Audits/blob/main/KISHIELD_MinionsINU_Audit_20230117.pdf
⭐MinionsINU live on CMC:
https://coinmarketcap.com/currencies/minionsinu/
✅Other important links:
⚛️DexTools Chart(Logo,Website&Socials updated): https://www.dextools.io/app/en/bnb/pair-explorer/0x675b21e2339291BdA903dD3Ed2246879c61Ed879
✅Liquidity Locked until 2024: https://www.pinksale.finance/pinklock/record/1065725?chain=BSC
✅WhitePaper: https://minionsinu.net/whitepaper.pdf
✅We bought the website domain via GoDaddy for 5 years.: https://who.is/whois/minionsinu.net
%50 Supply burnt forever before launch: https://bscscan.com/token/0xba7e2a9e5193e60368f440e4ae881cc312d6a160?a=0x000000000000000000000000000000000000dead
⭐First Burnt Evet: We burned all the tokens we had in the remaining DEV wallet.
We will finance MinionsINU with the proceeds from our NFT sales, our Betting Platform, and investments in our MINIONSVERSE world.
https://bscscan.com/tx/0xfa3f0b983cb8158541210038fbe650614a32f0fca3e76233563de842405436af
⭐MinionsINU most shared Posts on Instagram under #TrustWallet tag, you can check via Instagram or we uploaded screenshots:
https://imgur.com/s0vVqq6
Team and Community always active on Twitter Space:
Team with BNB chain: https://imgur.com/wYnX9VQ
Team with ASTAR NETWORK and SOTA as speaker: https://imgur.com/9MiPYfN
Team with Benzinga and RYO Speaker: https://imgur.com/x7nB7Lh
Team with BabyDogeCoin: https://imgur.com/jBGmDM6
Team with CryptoComNFT: https://imgur.com/JYUEsJN

Please review the details and let us know if payment is required. Best regards and good work, MinionsINU Team

**Comment by CryptoDevilJapan** [comment]
If you see fit, we would like to make the payment immediately. We will be waiting for your approval. Respects.

File: blockchains/smartchain/assets/0xba7E2a9E5193E60368F440E4Ae881cC312d6a160/info.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "name": "twitter",
+            "url": "https://twitter.com/MinionsINUbsc"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/MinionsINUbsc"
+        },
+        {
+            "name": "youtube",
+            "url": "https://www.youtube.com/@minionsinu"
```
Comment (cryptomanz1) [inline blockchains/smartchain/assets/0xba7E2a9E5193E60368F440E4Ae881cC312d6a160/info.json:22]: you need remove ```www```

Comment (CryptoDevilJapan) [reply blockchains/smartchain/assets/0xba7E2a9E5193E60368F440E4Ae881cC312d6a160/info.json:22]: okey i ll try to remove it

Comment (CryptoDevilJapan) [reply blockchains/smartchain/assets/0xba7E2a9E5193E60368F440E4Ae881cC312d6a160/info.json:22]: I try to delete but i guess there is a mistake. I ll recreate new request for fix it.

**Pull request closed without merging**